odfuzz/generators.py

```diff
@@ -199,11 +202,14 @@ def generate(generator_format='uri'):
 class EdmInt64:
     @staticmethod
     def generate(generator_format='uri'):
-        value = str(random.randint(-9223372036854775808, 9223372036854775807)) + 'L'
-        if generator_format == 'uri' or generator_format == 'body':
-            return value
+        value_uri = str(random.randint(-9223372036854775808, 9223372036854775807)) + 'L'
+        value_body = str(random.randint(-9223372036854775808, 9223372036854775807))
```

Comment:
Are you REALLY want to send different random number to the URL than to the http body? Would be better to 
`value_uri = value_body + 'L'`